UnityProject/Assets/Scripts/Items/Storage/PizzaBox.cs

```diff
@@ -135,7 +135,7 @@ private void CloseBox()
 
 		protected override void Detonate()
 		{
-			Chat.AddLocalMsgToChat("<color=red>The pizza bomb violently explodes!</color>", gameObject);
+			Chat.AddActionMsgToChat(gameObject, "<color=red>The pizza bomb violently explodes!</color>");
```

Comment:
this one should be a combat message, I think

Reply:
Changed to be combat text. Also made it large because it's a bomb. It's now the same size as limb severing text.